### DISCUSSION TITLE
3  DL Primer – Machine Learning Systems

### DISCUSSION THREAD

**Discussion by giscus** [harvard-edge/cs249r_book#428] [Chapter Discussions]
# 3  DL Primer – Machine Learning Systems

https://mlsysbook.ai/contents/dl_primer/dl_primer.html

**Comment by fabiog1901** [comment]
I have 2 items to suggest:

- It would be great to have a proceeding section on Traditional ML, just so the learner knows where modern ML using deep learning has evolved from. The scikit-learn doc website could be a great source for coding examples.
- I humbly suggest we replace the example involving the boston_housing dataset with an ethical one. I'm sure we're not short of good examples!

**Reply by profvjreddi** [reply]
Hi @fabiog1901 -- thank you so much for taking the time to drop your feedback. Really appreciate it. 

> * It would be great to have a proceeding section on Traditional ML, just so the learner knows where modern ML using deep learning has evolved from. The scikit-learn doc website could be a great source for coding examples.

That's a good suggestion. I am reworking this chapter (along with the others as part of a major revision), so we will be sure to update it during that process. 

> * I humbly suggest we replace the example involving the boston_housing dataset with an ethical one. I'm sure we're not short of good examples!

I appreciate the candid feedback. Will do!

**Comment by profvjreddi** [comment]
Updated these in the new chapters 3 and 4! Thanks for the feedback.
